packages/reanalyze/reanalyze.2.23.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Dead values/types, exception, and termination analysis for OCaml/ReScript"
+description: """
+Experimental analyses for ReScript/OCaml/Reason: for globally dead values/types, exception analysis, and termination analysis.
+"""
+maintainer: ["Cristiano Calcagno"]
+authors: ["Cristiano Calcagno"]
+license: "MIT"
+homepage: "https://github.com/rescript-association/reanalyze"
+bug-reports: "https://github.com/rescript-association/reanalyze/issues"
+depends: [
+  "dune" {>= "1.7"}
+  "ocaml" {>= "4.06.1" & < "4.15"}
```

Comment:
```suggestion
  "ocaml" {>= "4.08" & < "4.15"}
```
For [Unbound constructor Format.String_tag)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/8b788b03c6ef4115193a3b022d1d96fec7aa78ed/variant/opam-2.1,compilers,4.07,reanalyze.2.23.0)